kahuna/public/js/search-query/query-syntax.js

```diff
@@ -17,3 +18,7 @@ export function removeLabel(q, label) {
 export function removeLabels(q, labels) {
     return labels.reduce((q, curr) => removeLabel(q, curr.name), q);
 }
+
+export function     getCollection(path) {
```

Comment:
I think this got fixed elsewhere. Does master need merged in so changes are reflected?